python/config.cpp

```diff
@@ -1,9 +1,50 @@
-#include <pybind11/pybind11.h>
+#include <arbor/version.hpp>
+
+#include <sstream>
+#include <iomanip>
+#include <ios>
 
-#include "config.hpp"
+#include <pybind11/pybind11.h>
+#include <pybind11/stl.h>
 
 namespace pyarb {
+    
+// Create and return a dictionary
```

Comment:
Say what the dictionary is, i.e. something like "returns a python dictionary that python users can use to look up which options the Arbor library was configured with at compile time."

Reply:
Done